testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -61,31 +62,9 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I see "SUSE Linux Enterprise Server 15 SP4 x86_64" product has been added
     Then the SLE15 SP4 product should be added
 
-@scc_credentials
 @uyuni
```

Comment:
a bit sad to switch from ui testing to command line?

probably a matter of tastes...

Reply:
True, but this is the only way for adding those channel to an Uyuni server. I strictly followed the documentation here.

Reply:
There is no UI for this task